dataframe/nullable_vector.gen.go

```diff
@@ -6,6 +6,8 @@ package dataframe
 
 import "time"
 
+//Uint8o:Uint8enerate uint8enny -in=$GOFILE -out=nullable_vector.Uint8en.Uint8o uint8en "Uint8=uint8,uint16,uint32,uint64,int8,int16,int32,int64,float32,float64,strinUint8,bool,time.Time"
```

Comment:
This line has been derived from:
`go:generate genny -in=$GOFILE -out=nullable_vector.gen.go gen "g=uint8,uint16,uint32,uint64,int8,int16,int32,int64,float32,float64,string,bool,time.Time"`
by replacing `g` with each type.
It's not so important but maybe we could consider rename `g` to something else.

Reply:
I think that `go-generate` lines should not be included in the generated file.
According to the [README](https://github.com/cheekybits/genny/blob/master/README.md#go-generate):

> The //go:generate line will be removed from the output

I guess that in our case are there because of the `g` replacement.

Reply:
Doesn't impact this. But I renamed `g` to `gen` so it is a little bit easier to read in the two template files.